[2022-07-28 21:45:10,107] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.dag manual__2022-07-29T01:45:05.559178+00:00 [queued]>
[2022-07-28 21:45:10,111] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.dag manual__2022-07-29T01:45:05.559178+00:00 [queued]>
[2022-07-28 21:45:10,112] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 21:45:10,112] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 21:45:10,112] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 21:45:10,125] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): dag> on 2022-07-29 01:45:05.559178+00:00
[2022-07-28 21:45:10,126] {standard_task_runner.py:52} INFO - Started process 13610 to run task
[2022-07-28 21:45:10,128] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'dag', 'manual__2022-07-29T01:45:05.559178+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/mydag.py', '--cfg-path', '/tmp/tmpkycsdbjr', '--error-file', '/tmp/tmpfo_qfl1p']
[2022-07-28 21:45:10,128] {standard_task_runner.py:80} INFO - Job 29: Subtask dag
[2022-07-28 21:45:10,161] {task_command.py:370} INFO - Running <TaskInstance: my_dag.dag manual__2022-07-29T01:45:05.559178+00:00 [running]> on host DESKTOP-5GAFI6N.localdomain
[2022-07-28 21:45:10,198] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=dag
AIRFLOW_CTX_EXECUTION_DATE=2022-07-29T01:45:05.559178+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-29T01:45:05.559178+00:00
[2022-07-28 21:45:10,199] {logging_mixin.py:115} INFO - hi there!
[2022-07-28 21:45:10,199] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 21:45:10,203] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=dag, execution_date=20220729T014505, start_date=20220729T014510, end_date=20220729T014510
[2022-07-28 21:45:10,220] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 21:45:10,235] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
